pyqtgraph/parametertree/parameterTypes/action.py

```diff
@@ -1,8 +1,10 @@
 from ...Qt import QtCore, QtWidgets, QtGui
+from ...functions import disconnect
```

Comment:
@j9ac9k do we have `isort` guidelines for import ordering? I.e. case-sensitivity and all that? I figure it's out of scope for this PR (lots of noncompliant files throughout the lib), but something to consider for future contributions

Reply:
We do... but they're not enforced (and I don't plan on enforcing them for now); I ran isort on the entire repo a while back to mostly get it in good shape, identify bugs, and pain points and address them accordingly.  I have isort as part of my pre-commit hooks, but we don't mandate contributors use pre-commit yet.  I figure we'll mandate this once we can run black/pycln/isort when the PR queue is non-existent and in turn enable pre-commit.ci